block-storage/src/main/scala/coop/rchain/blockstorage/BlockDagFileStorage.scala

```diff
@@ -218,6 +235,8 @@ final class BlockDagFileStorage[F[_]: Concurrent: Sync: Log: RaiseIOError] priva
       } else {
         false.pure[F]
       }
+    def lookupByDeployId(deployId: DeployId): F[Option[BlockHash]] =
+      blockHashesByDeploy.get(deployId).pure[F]
     def topoSort(startBlockNumber: Long): F[Vector[Vector[BlockHash]]] =
```

Comment:
nit: please always leave whitespace between methods. It's hard to read them and scan for them otherwise.